tests/test_notebook.py

```diff
@@ -7,23 +7,24 @@
 
 
 this_dir = os.path.dirname(os.path.abspath(__file__))
-
+TESTS_NB_FILE = os.path.abspath(os.path.join(os.path.dirname(__file__), 'misc', 'memestra_nb_demo.ipynb'))
+import sys
```

Comment:
I thought you would be removing that hack?

Reply:
Sorry forgot to clarify this
No, it's not running on the  CI
The tests fail just as they fail on my computer